packages/cli/src/lib/commands/hub.ts

```diff
@@ -20,6 +20,11 @@ export const hub: CommandDefinition = (program) => {
         .action(async () => {
             const space = getConfig().lastSpaceId;
 
+            if (!space) {
+                console.error("No space selected");
+                process.exit(1);
```

Comment:
I'm not sure if we should finish commands with exit. For future use if we will have some clean up at the end of program, this line would skip it, and it can be easily handled in different way.